src/Appwrite/Platform/Workers/Hamster.php

```diff
@@ -0,0 +1,454 @@
+<?php
+
+namespace Appwrite\Platform\Workers;
+
+use Appwrite\Usage\Stats;
+use Appwrite\Event\Event;
+use Appwrite\Event\Hamster as EventHamster;
+use Appwrite\Network\Validator\Origin;
+use Utopia\Analytics\Adapter\Mixpanel;
+use Utopia\Analytics\Event as AnalyticsEvent;
+use Utopia\App;
+use Utopia\Cache\Cache;
+use Utopia\CLI\Console;
+use Utopia\Config\Config;
+use Utopia\Platform\Action;
+use Utopia\Database\Database;
+use Utopia\Database\Query;
+use Utopia\Database\Validator\Authorization;
+use Utopia\Database\Document;
+use Utopia\Queue\Message;
+use Utopia\Logger\Log;
+use Utopia\Pools\Group;
+
+class Hamster extends Action
+{
+    private array $metrics = [
+        'usage_files' => 'files.$all.count.total',
+        'usage_buckets' => 'buckets.$all.count.total',
+        'usage_databases' => 'databases.$all.count.total',
+        'usage_documents' => 'documents.$all.count.total',
+        'usage_collections' => 'collections.$all.count.total',
+        'usage_storage' => 'project.$all.storage.size',
+        'usage_requests' => 'project.$all.network.requests',
+        'usage_bandwidth' => 'project.$all.network.bandwidth',
+        'usage_users' => 'users.$all.count.total',
+        'usage_sessions' => 'sessions.email.requests.create',
+        'usage_executions' => 'executions.$all.compute.total',
+    ];
+
+    protected string $directory = '/usr/local';
+
+    protected string $path;
+
+    protected string $date;
+
+    protected Mixpanel $mixpanel;
+
+    public static function getName(): string
+    {
+        return 'hamster';
+    }
+
+    /**
+     * @throws \Exception
+     */
+    public function __construct()
+    {
+        $this->mixpanel = new Mixpanel(App::getEnv('_APP_MIXPANEL_TOKEN', ''));
```

Comment:
Lets move this initialisation to the `action()` function and also throw an exception if the mixpanel token is missing